isobus/src/can_network_manager.cpp

```diff
@@ -39,6 +39,15 @@ namespace isobus
 		{
 			get_next_can_message_from_tx_queue();
 		}
+
+		//! @todo: Remove this unsafe way of making a shared pointer to the network manager once the network manager is no longer a singleton,
```

Comment:
Double-free is rough... might be nicer to add a raw pointer version in the short term that we then remove once we ditch the singleton? Could make this raw pointer function badged or use a friend or something to keep it away from others if we wanted to.